### DISCUSSION TITLE
ChatList - Lobe UI

### DISCUSSION THREAD

**Discussion by giscus** [lobehub/lobe-ui#124] [Q&A]
# ChatList - Lobe UI

ChatList is a component used to display a list of chat messages. It takes in an array of ChatMessage objects as the data prop and an optional type prop which can be either 'docs' or 'chat'. The component is memoized for better performance.

https://ui.lobehub.com/components/chat-list

**Comment by zhangYYuan** [comment]
你好呀

**Reply by llz7sea** [reply]
Nothing to preview

**Comment by zhangYYuan** [comment]
你好

**Reply by zhangYYuan** [reply]
啦啦啦啦啦

**Comment by zhangYYuan** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by zhangYYuan** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
